src/PageCache/DefaultRequestPolicy.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Drupal\ssofact\PageCache;
+
+use Drupal\ssofact\PageCache\RequestPolicy\Cookie;
+use Drupal\Core\Session\SessionConfigurationInterface;
+use Drupal\Core\PageCache\DefaultRequestPolicy as PageCacheRequestPolicy;
+
+/**
+ * The ssofact page cache request policy.
+ *
+ * Caching is only allowed for requests without a session cookie.
+ */
```

Comment:
The summary line should summarize what the class is or is doing.  The current one is not really telling anything.  Try this:
```php
/**
 * Disallows serving responses from page cache for requests with a ssoFACT session cookie.
 */
```

EDIT: After reading through all changes, I now understand that this is actually overriding the default service implementation – that would have been clear right from the start if it would have stated so:
```php
/**
 * Overrides the default page cache policy service to add a custom rule for ssoFACT session cookies.
 *
 * Disallows serving responses from page cache for requests with a ssoFACT session cookie.
 */
```
…and the second line (PHPDoc description) probably fits better as a replacement for the PHPDoc summary line on the SessionCookieRequestPolicy class then.